yt_dlp/extractor/kanal2.py

```diff
@@ -0,0 +1,122 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from datetime import datetime
+import re
+import time
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    url_or_none,
+    traverse_obj,
+)
+
+
+class Kanal2IE(InfoExtractor):
+    SUBTITLE_DATE_RE = re.compile(r'\((\d{2}\.\d{2}\.\d{4}\s\d{2}:\d{2})\)$')
+
+    _VALID_URL = r'https?://.+\.postimees\.ee/[^?]+\?(.*?&)?id=(?P<id>\d+)'
+    _TESTS = [
+        {
+            'note': 'Test standard url (#5575)',
+            'url': 'https://kanal2.postimees.ee/pluss/video/?id=40792',
+            'md5': 'baf49cab468cf553bc1cf36909f1a0c7',
+            'info_dict': {
+                'id': '40792',
+                'ext': 'mp4',
+                'title': 'Aedniku aabits / Osa 53  (05.08.2016 20:00)',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'description': 'md5:53cabf3c5d73150d594747f727431248',
+                'upload_date': '20160805',
+                'timestamp': 1470416400,
+            }
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        playlist = self.get_playlist(video_id)
+
+        info = {
+            'id': video_id,
+            'title': self.get_title(playlist.get('info')),
+            'description': traverse_obj(playlist, ('info', 'description')),
+            'webpage_url': traverse_obj(playlist, ('data', 'url')),
+            'thumbnail': traverse_obj(playlist, ('data', 'image')),
+            'formats': self.get_formats(playlist, video_id),
+            'timestamp': self.get_timestamp(traverse_obj(playlist, ('info', 'subtitle'))),
+        }
+
+        return info
+
+    def get_title(self, info):
+        title = info['title']
+
+        if info['subtitle']:
+            title += ' / ' + info['subtitle']
+
+        return title
```

Comment:
`info['subtitle']` is fatal and should not be. one-time use function, move into `_real_extract`
```py
        title = traverse_obj(playlist, ('info', 'title'))
        sub = traverse_obj(sub, ('info', 'subtitle'))
        if sub:
            title = f'{title} / {sub}'
```
`sub` var can be used for the `timestamp` too

Reply:
```suggestion
    title = join_nonempty(*traverse_obj(info, ('info', ('title', 'subtitle'))), delim=' / ')
```